libwallet/src/slate.rs

```diff
@@ -260,6 +273,20 @@ impl Slate {
 			true => SlateState::Invoice1,
 			false => SlateState::Standard1,
 		};
+		let kernel_features = if kernel_feat_param > 3 {
+			0
```

Comment:
my guess is that `kernel_feat_param` being > 3 means there's a bug, so is it possible that a panic would be better in this case?

Reply:
agreed, fixed